### PR TITLE
Updated SharePoint App bar article

### PR DESCRIPTION
**Pull request by ganesh-sanap**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/sharepoint-app-bar.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ The SharePoint app bar brings together intranet resources and personalized conte
 > -	The SharePoint app bar may impact current page customizations specifically those that appear on the left side. 
 > -	Personalized content in the SharePoint app bar is enabled by Microsoft Graph.
 > -	When Microsoft Graph is disabled, the news and sites experience will be degraded.
-> -	The SharePoint app bar can be temporarily disabled between today and when it becomes available to all customers to give customers more time to prepare for this change. Temporarily disabling the ap bar will delay the roll out of this feature in your organization until October 31, 2021.
+> -	The SharePoint app bar can be temporarily disabled between today and when it becomes available to all customers to give customers more time to prepare for this change. Temporarily disabling the app bar will delay the roll out of this feature in your organization until October 31, 2021.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sharepoint-app-bar.md:44]:
```suggestion
> -	The SharePoint app bar can be temporarily disabled between today and when it becomes available to all customers to give customers more time to prepare for this change. Temporarily disabling the app bar will delay the rollout of this feature in your organization until October 31, 2021.
```

File: SharePoint/SharePointOnline/sharepoint-app-bar.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ Global navigation logo specifications:
 
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;![Image of where the logo and title will appear in global navigation](media/app-bar-title-logo.png)
 
-7.	Finally, determine the **Navigation source**. Learn more about selecting a source.
+7.	Finally, determine the **Navigation source**. Learn more about selecting a source below.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sharepoint-app-bar.md:88]:
```suggestion
7.	Finally, determine the **Navigation source**. Learn more about selecting a source in the next step.
```

File: SharePoint/SharePointOnline/sharepoint-app-bar.md [hunk 3]
```diff
@@ -114,8 +114,8 @@ If you haven’t set up your [home site](https://docs.microsoft.com/sharepoint/h
 
 ![Image of site navigation](media/app-bar-site.png)
 
-- Select the site navigation source to display the home site navigation.
-- Create a secondary set of navigation notes specifically for the global navigation panel by selecting **Hub or global navigation**. Then, select **Edit global navigation** to create the new global navigation menu. Select **Save** when you are done.
+- Select the site navigation source to display the home site’s navigation.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sharepoint-app-bar.md:117]:
```suggestion
- Select the site navigation source to display the home site's navigation.
```

File: SharePoint/SharePointOnline/sharepoint-app-bar.md [hunk 4]
```diff
@@ -176,12 +176,12 @@ If you are creating a [home site](https://docs.microsoft.com/sharepoint/home-sit
...
-6.	For the global navigation source, select **Home site navigation** to display the home site navigation that you created in the global navigation panel. Then select **Save**.
+6.	For the global navigation source, select **Home site navigation** to display the home site navigation that you created in the global navigation panel, then select **Save**.
 
 ## Understand how the app bar may impact page customizations
 
 The SharePoint app bar may impact current page customizations, specifically those positioned to the left of your page. 
-For organization using [page placeholders](https://docs.microsoft.com/sharepoint/dev/spfx/extensions/get-started/using-page-placeholder-with-extensions), the SharePoint app bar will cover parts of both the header and footer page placeholder. In the following image, the place holder footprint is in red:
+For organization using [page placeholders](https://docs.microsoft.com/sharepoint/dev/spfx/extensions/get-started/using-page-placeholder-with-extensions), the SharePoint app bar will cover parts of both the header and footer page placeholder. In the following image, the placeholder footprint is in red:
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sharepoint-app-bar.md:184]:
```suggestion
For organizations using [page placeholders](https://docs.microsoft.com/sharepoint/dev/spfx/extensions/get-started/using-page-placeholder-with-extensions), the SharePoint app bar will cover parts of both the header and footer page placeholder. In the following image, the placeholder footprint is in red:
```

File: SharePoint/SharePointOnline/sharepoint-app-bar.md [hunk 5]
```diff
@@ -148,8 +148,8 @@ Display just the hub navigation on the home page, and the site navigation in the
 1.	Start by hiding the site navigation using one of two methods:
 <br>
 
-- Go to **Setting** then **Change the look**, then **Navigation** and toggle the **Display site navigation** to **Hide**
-- Go to **Setting**, then **Change the look**, then **Header** and choose **Extended**
+- Go to **Settings**, then **Change the look**, then **Navigation** and toggle the **Display site navigation** to **Hide**
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sharepoint-app-bar.md:151]:
```suggestion
- Go to **Settings**, then **Change the look**, then **Navigation** and toggle the **Display site navigation** to **Hide**.
```

File: SharePoint/SharePointOnline/sharepoint-app-bar.md [hunk 6]
```diff
@@ -148,8 +148,8 @@ Display just the hub navigation on the home page, and the site navigation in the
...
+- Go to **Settings**, then **Change the look**, then **Header** and choose **Extended** layout
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sharepoint-app-bar.md:152]:
```suggestion
- Go to **Settings**, then **Change the look**, then **Header** and choose **Extended** layout.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - a few minor edits, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Holland-ODSP PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**